cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -50,28 +58,35 @@
       </collision>
     </link>
 
+    <joint name="${name}_charger_joint" type="fixed">
+      <origin xyz="0.2992 0.0 0.09089" rpy="0.0 0.29 0.0" />
```

Comment:
I think rpy should be 0 0 0. This is wrong in @ipa-tif's table. It's probably the angle which points up from base_link but we want the chrager link to be alligned with the floor

Reply:
rpy = (0 0 0)